src/compas_fab/robots/robot.py

```diff
@@ -979,24 +979,24 @@ def inverse_kinematics(self, frame_WCF, start_configuration=None, group=None, op
         start_configuration: :class:`compas_fab.robots.Configuration`, optional
             If passed, the inverse will be calculated such that the calculated
             joint positions differ the least from the start_configuration.
-            Defaults to the init configuration.
+            Defaults to the zero configuration.
         group: str, optional
             The planning group used for calculation. Defaults to the robot's
             main planning group.
+        return_full_configuration : bool, optional
+            If ``True``, returns a full configuration with all joint values
+            specified, including passive ones if available. Defaults to ``False``.
         options: dict, optional
             Dictionary containing the following key-value pairs:
 
             - avoid_collisions :: bool, optional
```

Comment:
Here the documentation isn't in the correct style, but it may have already been fixed on the main branch.